tests/eosio.system_tests.cpp

```diff
@@ -3546,6 +3544,32 @@ BOOST_FIXTURE_TEST_CASE(genesis_rewards, eosio_system_tester) try {
    BOOST_REQUIRE_EQUAL(get_balance(N(user11111111)), prev_balance + rewards_1_day);
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(backward_rewards, eosio_system_tester) try {
+   cross_15_percent_threshold();
+
+   create_account_with_resources(N(user11111111), config::system_account_name, core_sym::from_string("100.0000"), false,
+                                 core_sym::from_string("10.0000"), core_sym::from_string("10.0000"));
+
+   // This transer creates a sub_balance for genesis.wax account
+   transfer( N(eosio), N(genesis.wax), core_sym::from_string("200000000.0000"), N(eosio) );
+
+   uint64_t nonce = 1;
+   asset genesis_tokens_user1{core_sym::from_string("100000000.0000")};   
+
+   produce_blocks(1);
```

Comment:
maybe this one is redundant, given produce_block below

Reply:
Even though this seems to be redundant, there is a different between `produce_blocks()` (to produce n block) and `produce_block()` (to produce a block with the specified time-stamp) that makes required to call the first one first. Otherwise we get the _3040005 expired_tx_exception: Expired Transaction_ error.

Reply:
ahhhhh. It's like a really long block which would cause the expiration. Thanks Ernesto.